junwoo/controllers/likeController.js

```diff
@@ -0,0 +1,11 @@
+const likeService = require("../services/likeService");
+
+const likeUp = async (req, res) => {
+  const { userId, postId } = req.body;
```

Comment:
`userId`와 `postId`는 데이터베이스에 필수적으로 추가가 되어야하기 때문에 해당 데이터가 없을 경우 에러메시지를 반환하도록 예외처리를 해주세요!
```suggestion
  const { userId, postId } = req.body;
```
```suggestion
  const { userId, postId } = req.body;
  
  if( !userId| !postId) {
    throw new Error("Key Error!");
  }
```